build/azure-pipelines.yml

```diff
@@ -1,63 +1,67 @@
 # Repo: FirelyTeam/Vonk.Plugin.DocumentOperation
 # File: build/azure-pipelines.yml
-name: build-$(Date:yyyyMMdd)$(Rev:-r)
-
+name: build-$(Date:yyyyMMdd)$(Rev:.r)
 resources:
   repositories:
     - repository: templates
       type: github
       name: FirelyTeam/azure-pipeline-templates
       endpoint: FirelyTeam
       ref: refs/tags/v1
-      
 
 variables:
-  buildConfiguration: 'Release'
+  - template: variables.yml  
+
+pool: 
+  vmImage: $(vmImage)
 
 trigger:
-  branches:
+  branches: 
     include:
-    - develop*
-    - release*
-
-pr:
-  branches:
-    include: 
-    - develop*
+    - develop
+    - release/*
     - hotfix/*
+    - refs/tags/v*
```

Comment:
Not sure if this is relevant.

Reply:
Yes it is, because now this pipeline should be triggered by a release as well. This was first taken care of the other (release) pipeline, which has been removed now.

Reply:
Oh, I got confused, I thought this was for the `pr` section.

Reply:
But the `pr` section disappeared. Is there a reason for that?

Reply:
But I can bring it back if you want.